src/ADAL.NET/AuthenticationContext.cs

```diff
@@ -774,8 +793,23 @@ public AuthenticationResult AcquireTokenSilent(string resource, ClientAssertion
         /// <returns>URL of the authorize endpoint including the query parameters.</returns>
         public Uri GetAuthorizationRequestURL(string resource, string clientId, Uri redirectUri, UserIdentifier userId, string extraQueryParameters)
         {
-            var handler = new AcquireTokenInteractiveHandler(this.Authenticator, this.TokenCache, resource, clientId, redirectUri, PromptBehavior.Auto, userId, extraQueryParameters, null, true);
-            return RunAsyncTask(handler.CreateAuthorizationUriAsync(this.CorrelationId));            
+            return GetAuthorizationRequestURL(resource, clientId, redirectUri, userId, extraQueryParameters, null);
+        }
+
+        /// <summary>
+        /// Gets URL of the authorize endpoint including the query parameters.
+        /// </summary>
+        /// <param name="resource">Identifier of the target resource that is the recipient of the requested token.</param>
+        /// <param name="clientId">Identifier of the client requesting the token.</param>
+        /// <param name="redirectUri">Address to return to upon receiving a response from the authority.</param>
+        /// <param name="userId">Identifier of the user token is requested for. This parameter can be <see cref="UserIdentifier"/>.Any.</param>
+        /// <param name="extraQueryParameters">This parameter will be appended as is to the query string in the HTTP authentication request to the authority. The parameter can be null.</param>
+        /// <param name="claims">Additional claims that are needed for authentication. Acquired from the AdalClaimChallengeException. This parameter can be null.</param>
```

Comment:
claims documentation is incorrect. please see adal v3 and make necessary changes everywhere.